django/cantusdb_project/main_app/views/chant.py

```diff
@@ -1389,9 +1389,17 @@ def get_context_data(self, **kwargs):
 
         # 4064 is the id for the sequence database
         if source.segment.id == 4064:
-            queryset = source.sequence_set.order_by("folio", "s_sequence")
+            queryset = source.sequence_set.annotate(
+                is_sequence=Value(True)
+            ).annotate(
+                is_chant=Value(False)
+            ).order_by("s_sequence")
```

Comment:
I see how this fixes the issue in #583. 

For chants, might you run into this same problem, caused by the way folios are sorted without the leading 0? I see Andrew's comment about natural sorting on that issue. 

Probably out of scope for this PR, but I'm curious if that's an issue.

Reply:
Having worked with manuscripts for a while now, the one thing I'm certain of is that there is no consistency. 

 - Some MSS have +999 pages, so three digits are not enough.
 - Some folios are inserts or flyouts, so they may have an additional qualifiers (a/b) to indicate that they're different. 
 - Many books use pagination instead of foliation (p1, p2, p3 instead of 001r, 001v, 002r)
 - Many books were re-ordered, sometimes even recently, so they might have two folio numbers -- the original and the new. 
 - Many books were re-foliated. This may occur if someone mis-counted and skipped a folio, so someone had to come along later and fix this. Or double-counted.
 - Some books have inserts that are numbered outside of the "main" numbering / foliation, e.g., i, ii, iii, iv, ix, etc. 
 - Some books have been split in two, and the foliation continues in the new book, so they can start at f333r, etc.
 - Some books have been joined into one, so they can have their foliation re-started halfway through!

Many of these may contradict the editorial standards of Cantus, but it's not necessarily a given that all MSS will behave the same way. So I would be careful about trying to standardize their form too much...

Reply:
In fact, the Salzinnes is an example of a book which has two numbering schemes:

https://cantus.simssa.ca/manuscript/25/?folio%5B0%5D=A22r&pageAlias=Folio%20A22r

fA22r is part of the second half of the book.

Reply:
So all of this amounts to _folio numbering systems are too variable to try to standardize, so what we have in this PR is fine and let's not try to further tweak the ordering of results_, correct? No further action necessary?

Reply:
Approving, because I agree that we don't want to re-do sorting given these various manuscript-by-manuscript idiosyncrasies. 

I guess I just want to clarify what this PR does and does not "solve." Because of all that's mentioned above about folio naming/numbering, items on the full index page may end up out of order. Since `s_sequence` numbers work the way they do (by manuscript), we can use them with this minor change to correctly order the index page from here on. Because `c_sequence` numbers work differently (by folio), ensuring "proper" ordering would require a more complex change and the decision is not to do that at this point (or maybe ever). Right?

Reply:
> I guess I just want to clarify what this PR does and does not "solve." Because of all that's mentioned above about folio naming/numbering, items on the full index page may end up out of order. Since `s_sequence` numbers work the way they do (by manuscript), we can use them with this minor change to correctly order the index page from here on. Because `c_sequence` numbers work differently (by folio), ensuring "proper" ordering would require a more complex change and the decision is not to do that at this point (or maybe ever). Right?

All this is correct!